src/client/js/Panels/MetaEditor/MetaEditorControl.js

```diff
@@ -1174,46 +1174,71 @@ define(['js/logger',
     };
 
     MetaEditorControl.prototype._createPointerRelationship = function (sourceID, targetID, isSet) {
-        var sourceNode = this._client.getNode(sourceID),
-            targetNode = this._client.getNode(targetID),
+        var client = this._client,
+            sourceNode = client.getNode(sourceID),
+            targetNode = client.getNode(targetID),
+            metaNodes = client.getAllMetaNodes(true),
             pointerMetaDescriptor,
+            existingNames,
             existingPointerNames,
-            notAllowedPointerNames,
-            self = this;
+            notAllowedPointerNames;
+
+        function getExistingNames() {
```

Comment:
I changed this, now it only checks what's defined for the current node. If there are collisions down the inheritance chains - those issues are caught by the MetaEditor using #1258